app/main.py

```diff
@@ -1,3 +1,30 @@
 class OnlineCourse:
-    # write your code here
-    pass
+    DAYS_IN_WEEKS = 7
+
+    def __init__(self, name, description, weeks):
+        self.name = name
+        self.description = description
+        self.weeks = weeks
+
+    @staticmethod
+    def days_to_weeks(days):
+        if days < 0:
+            return 0
+        else:
+            return days // OnlineCourse.DAYS_IN_WEEKS \
+                + bool(days % OnlineCourse.DAYS_IN_WEEKS)
+
+    @classmethod
+    def from_dict(cls, dictionary):
+        if 'days' in dictionary:
+            return OnlineCourse(
```

Comment:
Please, use `cls` to create an instance in this method.
And use `OnlineCourse` to call staticmethods )

Reply:
The reasons will be discussed in OOP lessons )